wgpu-core/src/pipeline.rs

```diff
@@ -373,6 +373,8 @@ pub enum CreateRenderPipelineError {
         stage: wgt::ShaderStages,
         error: String,
     },
+    #[error("shader group {group} binding {binding} must be 16 bytes aligned as device does not support DownlevelFlags `BUFFER_BINDINGS_NOT_16_BYTE_ALIGNED`")]
```

Comment:
```suggestion
    #[error("In the provided shader, the type given for group {group} binding {binding} has an alignment of {alignment}. As the device does not support `DownlevelFlags::BUFFER_BINDINGS_NOT_16_BYTE_ALIGNED` the type must have an alignment of at least 16.")]
```

Sorry, one more round on this error message - note the extra argument alignment. Adding the eixsting alignment really helps make the message clear.

Reply:
@cwfitzgerald I'm trying to get the alignment, but I'm not sure how to go about this. Do I just get the size of the binding?